fah_alchemy/settings.py

```diff
@@ -1,3 +1,9 @@
+"""
+Settings --- :mod:`fah-alchemy.cli`
```

Comment:
I think this should be `fah-alchemy.settings`?

Reply:
Ah cheers for picking that up.